### PR TITLE
Chapter 10 part 2

### PR DESCRIPTION
**Pull request by ndviet2307**

### CODE REVIEW

**Comment by ndviet2307** [comment]
screenshot from 2018-09-12 13-11-53

**Comment by ndviet2307** [comment]
ready

**Comment by ndviet2307** [comment]
screenshot from 2018-09-12 14-17-59

**Comment by ndviet2307** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -19,8 +23,49 @@ def create
     end
   end
 
+  def show
+    redirect_to signup_path if @user.nil?
```
Comment (thanhhungchu95) [inline app/controllers/users_controller.rb]: unless @user

**Comment by ndviet2307** [comment]
screenshot from 2018-09-12 15-04-53

**Comment by ndviet2307** [comment]
ready

**Pull request merged**